docs/reference/ml/ml-shared.asciidoc

```diff
@@ -524,6 +524,36 @@ been reported.
 `peak_usage_bytes`::::
 (long) The number of bytes used at the highest peak of memory usage.
 
+`node`:::
+(object)
+Contains properties for the node that runs the job. This information is 
+available only for running jobs.
+
+`node`.`attributes`:::
```

Comment:
Although this is the format we use for anomaly detection, I think we're now mixing two different ways of listing nested fields.

We have:

```
`node`.`attributes`:::
```

and simply:

```
`peak_usage_bytes`::::
```

So, the first includes the parent field name and keeps things at the same level (3 colons). The other doesn't include the parent field and indents one more (4 colons). I think I prefer the latter but as long as we're consistent I don't mind so I'll leave it on you.

Reply:
@dimitris-athanasiou 
Yes, I spotted that. I would like to apply these changes to 7.6, so I didn't want to modify the changes that are applicable only for 7.x because that way it is harder to cherry-pick. I'll make the docs uniform in an upcoming PR.